sparkctl/cmd/client.go

```diff
@@ -21,12 +21,24 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	"os"
```

Comment:
Sorry I meant this should be in the group of golang built-in packages.

Reply:
You mean, "os" at the first line in import, then an empty line, and then the rest of the import packages?

Reply:
```import (
	"os"

	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	clientset "k8s.io/client-go/kubernetes"
	"k8s.io/client-go/rest"
	"k8s.io/client-go/tools/clientcmd"

	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io/v1beta1"
	crdclientset "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/clientset/versioned"
)```

Reply:
Yes

Reply:
:) Done